pages/team.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: page
+title: MDAnalysis team
+---
+
+MDAnalysis is a community-driven project that is made possible through the efforts of many members who contribute in numerous and diverse ways, ranging from direct package development, maintenance, documentation, communication, and managerial responsibilities. On this page we list identified project roles and those historically responsible for the tasks. In the future, we plan to also list the contributors currently responsible for overseeing and managing associated tasks. We note that the listed roles on this page can differ significantly in scope and required effort.
+
+## Roles
```

Comment:
How far back the historical roles go?

Reply:
I'm guessing 1-2 years? I've updated the page with these notes but not entirely certain what people were thinking as they filled out the spreadsheet!

Reply:
Not sure either, but I think it's good to specify. How you wrote it now sounds great IMO, thanks!

Reply:
Probably a bit more than that, I would say roughly 2020 - that would better coincide with some of the data we used for things like PR & issue comments.

Reply:
Updated!